docs/sources/features/datasources/elasticsearch.md

```diff
@@ -197,13 +197,8 @@ for annotation events.
 
 > Only available in Grafana v6.3+.
 
-Querying and displaying log data from Elasticsearch is available via [Explore]({{< relref "../../explore" >}}).
-
-![](/img/docs/v63/elasticsearch_explore_logs.png)
-
-Select the Elasticsearch data source, change to Logs using the Metrics/Logs switcher, and then optionally enter a lucene query into the query field to filter the log messages.
-
-Finally, press the `Enter` key or the `Run Query` button to display your logs.
+Querying and displaying log data from Elasticsearch is available via [Explore]({{< relref "../../explore" >}}), and with the [logs panel]({{< relref "../../panels/visualizations/logs-panel.md" >}}) in dashboards. 
```

Comment:
Suggestion: replace "via" with "in"

"and in the [logs panel] ..."

Reply:
I agree with replacing "via" with "in", but not sure about the logs panel because querying is done in the query editor which is not _in_ the logs panel. Rather, I imagine the semantics being that "displaying log data is available in dashboards _with_ the [use of] logs panel".

Reply:
Thank you for the explanation. Can you rephrase using either "through" or "from" whichever you think is a better fit? With is not the preposition to use here.

Reply:
On second thought, I think "in" would be the most appropriate here since it makes it clear that the logs panel is _where_ the functionality can be found!

Reply:
Thank you for taking the time to think over and making the documentation "just right" :)